SimTKmath/Optimizers/src/IpOpt/IpLapackSolverInterface.hpp

```diff
@@ -160,6 +160,7 @@ namespace Ipopt
     //MUMPS data structure
 //    DMUMPS_STRUC_C mumps_data;
 
+    int *ipiv_;
```

Comment:
For consistency, please move this up next to the other data members (below `negevals_`), with a comment like `/** Array for storing the pivot order after factorization. */`.